pkg/apis/pipeline/v1/pipeline_validation.go

```diff
@@ -497,6 +497,22 @@ func (pt *PipelineTask) extractAllParams() Params {
 	return allParams
 }
 
+// GetVarSubstitutionExpressions extract all values between the parameters "$(" and ")" of steps and sidecars
+func (pt *PipelineTask) GetVarSubstitutionExpressions() []string {
```

Comment:
please add tests for this exported function 

https://github.com/tektoncd/community/blob/main/standards.md#go-packages

Reply:
done~, GetVarSubstitutionExpressions will be tested in `TestNewResultReferenceTaskSpec`